Detections/SecurityEvent/UserAccountAddedToPrivlegeGroup_1h.txt

```diff
@@ -0,0 +1,30 @@
+// Name: User Account added to Built in Domain Local or Global Group
+// Description: User account was added to a privileged built in domain local group or global group such as the Enterprise Adminis, Cert Publishers or DnsAdmins
+// Be sure to verify this is an expected addition
+//
+// Id: a35f2c18-1b97-458f-ad26-e033af18eb99
+//
+// Severity: Medium
```

Comment:
given this is legit admin operatio, just one that might be abused/copied by an attacker, do you think Low might be more suitable?

Reply:
Changed to low